source/initial_composition/function.cc

```diff
@@ -26,9 +26,6 @@ namespace aspect
 {
   namespace InitialComposition
   {
-//    template <int dim>
-//    Function<dim>::Function ()
-//    {}
```

Comment:
good catch -- this could have been a separate pull request, by the way. the change is conceptually independent, and it is easier to review independent things independently